flow-report/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "composite": true,
     "outDir": "../.tmp/tsbuildinfo/flow-report",
-    "emitDeclarationOnly": true,
+    "emitDeclarationOnly": false,
```

Comment:
@brendankenny this was causing `yarn build-report` to emit an empty chunk for `standalone-flow`

Reply:
Yeah, really it's only needed if something else is going to consume this as a `references` dependency and we don't need transpiled js output. Shouldn't be necessary for `references` and it has real transpiled output anyways :)